app/helpers/provisioning_templates_helper.rb

```diff
@@ -72,4 +72,25 @@ def how_templates_are_determined
     (_("The final entry, Operating System default, can be set by editing the %s page.") %
      (link_to _("Operating System"), operatingsystems_path))).html_safe)
   end
+
+  def download_utility(utility)
```

Comment:
I'd vore for extraction of this method. It's not tied to neither views nor erb macros. If there is no better place, I'd say we can bare with a custom module with this method or maybe include it in general helpers such as https://github.com/theforeman/foreman/blob/develop/lib/foreman.rb

Since it's a frozen hash, it could be a constant instead of a method.

Reply:
Moved to `lib/foreman.rb` as constant